core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -5090,6 +5091,19 @@ private HepProgram getTransitiveProgram() {
         .checkUnchanged();
   }
 
+  /** Test case for <a href="https://issues.apache.org/jira/browse/CALCITE-5882">
+   * [CALCITE-5882] Compile-time evaluation of SPLIT function returns incorrect result</a>. */
+  @Test public void testSplit() {
```

Comment:
Is there a reason this test is in RelOptRules? My understanding is this test class is for optimization rules, I am not sure if a function bug test necessarily belongs here. Maybe the operator tests would work better? Let me know if I am missing something.